policy/modules/contrib/dbus.if

```diff
@@ -122,10 +122,16 @@ template(`dbus_role_template',`
 
 	auth_use_nsswitch($1_dbusd_t)
 
+	files_config_all_files($1_dbusd_t)
+
 	logging_send_syslog_msg($1_dbusd_t)
 
 	dontaudit $1_dbusd_t self:capability net_admin;
 
+	optional_policy(`
+		systemd_start_systemd_services($1_dbusd_t)
```

Comment:
I'd rather see the optional block base on modules alphabetical order.

Reply:
Sure, my bad. Now it should be fixed. :)